### PR TITLE
Suppress possible ASP tags usage warning

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Suppress possible ASP tags usage warning

### Fixed Issues (if relevant)
1. Fixes magento/magento2#31081

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
Hi @sergeynezbritskiy,
Does it seem like this PR isn't fixing https://github.com/magento/magento2/issues/31081, but cleans up the codebase to have more clear reports?

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by sergeynezbritskiy** [comment]
Hi @ihor-sviziev 
Yes, you're absolutely right. The purpose of this PR is just to reduce noise in the report. And there will be few more PRs within [#31081](https://github.com/magento/magento2/issues/31081)

File: app/code/Magento/Backend/view/adminhtml/templates/system/search.phtml [hunk 1]
```diff
@@ -4,7 +4,16 @@
...
 
-/** @var $block \Magento\Backend\Block\GlobalSearch */
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
+
+/** @var $block GlobalSearch */
+
+use Magento\Backend\Block\GlobalSearch;
+use Magento\Framework\Json\Helper\Data;
+
+/** @var Data $helper */
```
Comment (ihor-sviziev) [inline app/code/Magento/Backend/view/adminhtml/templates/system/search.phtml]:
```suggestion
use Magento\Backend\Block\GlobalSearch;
use Magento\Framework\Json\Helper\Data;

/** @var $block GlobalSearch */
/** @var Data $helper */
```

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml [hunk 2]
```diff
@@ -6,6 +6,7 @@
 ?>
 
 <?php
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
```
Comment (ihor-sviziev) [inline app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml:7]: Could you remove useless opening and closing PHP tags just after the copyright?

File: app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/options/type/file.phtml [hunk 3]
```diff
@@ -4,6 +4,7 @@
  * See COPYING.txt for license details.
  */
 
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
 ?>
 <?php /** @var $block \Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Options\Type\File */ ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/options/type/file.phtml]:
```suggestion
// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
/** @var $block \Magento\Catalog\Block\Adminhtml\Product\Edit\Tab\Options\Type\File */
?>
```

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/links.phtml [hunk 4]
```diff
@@ -5,6 +5,7 @@
  */
 
 // @deprecated
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
 ?>
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/links.phtml]:
```suggestion
// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound

```

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml [hunk 5]
```diff
@@ -5,6 +5,7 @@
  */
 
 // @deprecated
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
 ?>
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml]:
```suggestion
// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
```

File: app/code/Magento/Shipping/view/adminhtml/templates/order/tracking.phtml [hunk 6]
```diff
@@ -5,6 +5,7 @@
  */
 ?>
 <?php
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
 /**
```
Comment (ihor-sviziev) [inline app/code/Magento/Shipping/view/adminhtml/templates/order/tracking.phtml]:
```suggestion
// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
/**
```

File: app/code/Magento/Weee/view/adminhtml/templates/renderer/tax.phtml [hunk 7]
```diff
@@ -4,6 +4,7 @@
  * See COPYING.txt for license details.
  */
 
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
 ?>
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Weee/view/adminhtml/templates/renderer/tax.phtml]:
```suggestion
// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml:7]: fixed

File: app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/options/type/file.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/edit/options/type/file.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/links.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/links.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml]: fixed

File: app/code/Magento/Shipping/view/adminhtml/templates/order/tracking.phtml (hunk 6 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Shipping/view/adminhtml/templates/order/tracking.phtml]: fixed

File: app/code/Magento/Weee/view/adminhtml/templates/renderer/tax.phtml (hunk 7 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Weee/view/adminhtml/templates/renderer/tax.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Database Compare, Functional Tests B2B, Functional Tests EE

**Comment by ihor-sviziev** [comment]
Hi @sergeynezbritskiy,
Could you re-target this PR to the php8-develop branch if it is related to it?

**Comment by hostep** [comment]
This may be a stupid question, but could somebody explain what this `<%- ... %>` sytnax is? It's not knockout.js as far as I know, so what is this? And shouldn't we clean that up instead of ignoring it in coding standards?

**Comment by sergeynezbritskiy** [comment]
> This may be a stupid question, but could somebody explain what this `<%- ... %>` sytnax is? It's not knockout.js as far as I know, so what is this? And shouldn't we clean that up instead of ignoring it in coding standards?

Hi @hostep , looks like it is underscorejs syntax https://underscorejs.org/#template

**Comment by ihor-sviziev** [comment]
@magento run Database Compare, Functional Tests B2B, Functional Tests EE

**Comment by gabrieldagama** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Comment by sergeynezbritskiy** [comment]
> Hi @sergeynezbritskiy,
> Could you re-target this PR to the php8-develop branch if it is related to it?

Hi @ihor-sviziev, re-targeted to php8-develop

**Comment by sergeynezbritskiy** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**